### ISSUE TITLE
Formalize licensing policies

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/knowledge-graph-hub-support#5]
We can require new KG projects to have clearly defined license terms of CC0 (i.e., public domain) or CC-BY.
This can be included in a repo LICENSE file, but how can we verify its existence?

**Comment by matentzn** [comment]
Remember that you cant be cc0 if the upstream sources are not...

**Comment by caufieldjh** [comment]
True, and none of the data or the template project code are CC0, so that's not going to happen for a new project.

Licenses aren't tracked by the OBO Dashboard, are they?

**Comment by matentzn** [comment]
They are! Principle 1: http://dashboard.obofoundry.org/dashboard/index.html
